src/main/java/edu/kis/powp/jobs2d/drivers/DriverFeaturesManager.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.shape.line.BasicLine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.observer.Publisher;
+
+public class DriverFeaturesManager extends DriverManager {
```

Comment:
DriverFeaturesManager extends DriverManager, but it also seems to duplicate it (e.g., changePublisher).